content/chainguard/migration-guides/migrating-to-chainguard-images.md

```diff
@@ -0,0 +1,130 @@
+| Remove a package             | `apt remove`            | `apk del`              |
+| Update package manager cache | `apt update`            | `apk update`           |
+
+Our [Debian Compatibility](https://edu.chainguard.dev/chainguard/migration-guides/debian-compatibility/) page has a table listing common tools and their corresponding package(s) in both Wolfi and Debian distributions. For Ubuntu-based Dockerfiles, check our [Ubuntu Compatibility](https://edu.chainguard.dev/chainguard/migration-guides/ubuntu-compatibility/) page.
+
+## Migrating from Red Hat UBI Dockerfiles
+If you are coming from a Red Hat UBI (Universal Base Image) Dockerfile, you'll also need to adapt some of your commands to be compatible with the apk ecosystem. Wolfi uses BusyBox utilities, which offer a smaller footprint compared to GNU coreutils in Red Hat images. Our [Red Hat Compatibility](https://edu.chainguard.dev/chainguard/migration-guides/debian-compatibility/) page has a table listing common tools and their corresponding package(s) in both Wolfi and Red Hat distributions.
+
+If you are coming from a Red Hat UBI based Dockerfile, you'll need to adapt some of your commands to be compatible with the apk ecosystem:
+
+| Command Description          | Red Hat UBI Dockerfile | Wolfi-based Equivalent |
+|------------------------------|------------------------|------------------------|
+| Install a package            | `yum install`          | apk add                |
+| Remove a package             | `yum remove`           | apk del                |
+| Update package manager cache | `yum makecache`        | apk update             |
+
+## Migrating from Alpine Dockerfiles
+If your Dockerfile is based on Alpine, the process for migrating to Chainguard Images should be more straightforward, since you're already using `apk` commands. Wolfi packages typically match what is available in Alpine, with some exceptions. For instance, the Wolfi busybox package is slimmer and doesn't include all tools available in Alpine's busybox. Check the [Alpine Compatibility](https://edu.chainguard.dev/chainguard/migration-guides/alpine-compatibility/) page for a list of common tools and their corresponding packages in Wolfi and Alpine.
+
+## Searching for Packages
+Packages from Debian and other base distributions might have a different name in Wolfi. To search for packages, log into an ephemeral container based on `cgr.dev/chainguard/wolfi-base`:
+
+```shell
+docker run -it --rm --entrypoint /bin/sh cgr.dev/chainguard/wolfi-base
+```
+
+Then, run `apk update` to update the local apk cache with latest Wolfi packages:
+
+```shell
+# apk update
+fetch https://packages.wolfi.dev/os/x86_64/APKINDEX.tar.gz
+[https://packages.wolfi.dev/os]
+OK: 46985 distinct packages available
+```
+Now you can use `apk search` to look for packages. The following example searches for PHP 8.2 XML extensions:
+
+```shell
+# apk search php*8.2*xml*
```

Comment:
```suggestion
apk search php*8.2*xml*
```
Here especially since it's already separate from its output, I don' tthink the #'s are necessary